cli/cmd/plugin/conformance/e2e-test/e2e_test.go

```diff
@@ -0,0 +1,117 @@
+//go:build e2e
+
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e_test
+
+import (
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"io"
+	"math/big"
+	"os"
+	"os/exec"
+	"testing"
+)
+
+var tceRepoPath string
+var clusterName string
+var conformanceDirPath string
+var unmanagedClusterDirPath string
+var conformanceResultFilename string
+
+func initializeVariable() {
+	// Variables used across functions
+	tceRepoPath = os.Getenv("ROOT_DIR")
+	conformanceDirPath = tceRepoPath + "/cli/cmd/plugin/conformance"
+	unmanagedClusterDirPath = tceRepoPath + "/cli/cmd/plugin/unmanaged-cluster"
+	rand, _ := rand.Int(rand.Reader, big.NewInt(1000))
+	clusterName = "uc" + rand.String() + "conformance-test"
+}
+
+func TestSetup(t *testing.T) {
+	initializeVariable()
+
+	err := os.Chdir(unmanagedClusterDirPath)
+	if err != nil {
+		t.Errorf("error while changing directory to unmanged-cluster: %v", err)
+	}
+	// Creating Unmanaged cluster
+	_, err = cliRunner("go", nil, "run", ".", "create", clusterName)
+	if err != nil {
+		t.Errorf("Error while Unmanaged Cluster creation: %v", err)
+	}
+}
+
+func TestConformance(t *testing.T) {
+	err := os.Chdir(conformanceDirPath)
+	if err != nil {
+		t.Errorf("error while changing directory to conformance: %v", err)
+	}
+
+	// Running Conformance Plugin
+	_, err = cliRunner("go", nil, "run", ".", "run", "--wait")
+	if err != nil {
+		t.Errorf("Error while running Conformance plugin: %v", err)
+	}
+
+	// Retrieving Conformance result file
+	conformanceResultFilename, err = cliRunner("go", nil, "run", ".", "retrieve")
+	if err != nil {
+		t.Errorf("Error while retrieving the conformance result filename: %v", err)
+	}
+	conformanceResultFilename = "/" + conformanceResultFilename[:len(conformanceResultFilename)-1]
+
+	// Extracting the result
+	conformanceResult, err := cliRunner("go", nil, "run", ".", "results", conformanceDirPath+conformanceResultFilename)
+	if err != nil {
+		t.Errorf("Error while fetching the result from tarball produced by conformance: %v", err)
+	}
+	fmt.Printf("%v", conformanceResult)
```

Comment:
This all looks good to me but we should note that unmanaged cluster isn't targeted to be CNCF conformant and may have failures displayed in these results.

Reply:
Oh Okay so I am okay with not printing the result as that is not our main motive. What do you say?

Reply:
🤷🏼 

It could be useful to validate that _some_ results are being printed correctly (since that's one of the big value adds of the conformance plugin). But it's not super relevant to ensuring it works. So it's up to you!

Reply:
Yeah, agree with the value add by the result. Let’s have it then.